src/main/kotlin/com/vk/admstorm/startup/ChangeSshBackendStartup.kt

```diff
@@ -10,44 +10,50 @@ import com.vk.admstorm.notifications.AdmNotification
 object ChangeSshBackendStartup {
     private val LOG = logger<ChangeSshBackendStartup>()
 
-    private const val SSH_NOTIFICATOR_OPTION = "sshLegacyChanges"
+    private const val LEGACY_SSH_NOTIFICATOR_OPTION = "sshLegacyChanges"
     private const val SSH_CONFIG_BACKEND = "ssh.config.backend"
+    private const val SSH_NOTIFICATOR_OPTION = "sshOpenSSHChanges"
 
     private fun changeSshConfiguration(configValue: SshConnectionConfigService.Kind) {
         AdvancedSettings.setEnum(SSH_CONFIG_BACKEND, configValue)
     }
 
     fun changeConfigurationProcess(project: Project) {
+        //TODO: Remove it after 2 month
+        if (PropertiesComponent.getInstance(project).getValue(LEGACY_SSH_NOTIFICATOR_OPTION) != null) {
+            PropertiesComponent.getInstance(project).unsetValue(LEGACY_SSH_NOTIFICATOR_OPTION)
```

Comment:
We do `PropertiesComponent.getInstance(project)` 5 times. Let's do it 1 time.

Reply:
Fix